app/views/content/train-to-be-a-teacher/what-is-qts.md

```diff
@@ -68,7 +68,7 @@ If your teacher training course leads to QTS:
 As a teacher with QTS: 
 
 * you'll receive a package of [support for early career teachers](/life-as-a-teacher/teaching-as-a-career/early-career-teachers), including a reduced timetable 
-* you'll be on the qualified teacher pay range and receive a higher salary than teachers without QTS
+* you'll be on the qualified teacher pay range and [receive a higher salary than teachers without QTS](/life-as-a-teacher/pay-and-benefits/teacher-pay)
```

Comment:
There's something slightly odd going on with this bullet - the link starts on a new line for some reason (I can't see why) - might need some dev help with this!

![image](https://github.com/user-attachments/assets/c3fd3a0d-eb60-465e-86c5-72b8c88ba973)

![image](https://github.com/user-attachments/assets/734e84c8-13be-482b-8779-9472e599cebf)

Reply:
I think this is the bug with bullets - happens elsewhere on the page - we already have a ticket on the backlog about this so I will resolve this conversation!